src/index.ts

```diff
@@ -1036,6 +1065,8 @@ class Tinypool extends EventEmitterAsyncResource {
   }
 }
 
+const _workerId = process.__tinypool_state__?.workerId
+
 export * from './common'
-export { Tinypool, Options }
+export { Tinypool, Options, _workerId as workerId }
```

Comment:
So, the only way I can get worker id is by importing `tinypool` inside a script that is running in a worker? Like this:

```ts
import { workerId } from 'tinypool'
```

Reply:
or accessing `process.__tinypool_state__?.workerId`